purchase_request_to_rfq_order_approved/README.rst

```diff
@@ -0,0 +1,51 @@
+.. image:: https://img.shields.io/badge/licence-LGPL--3-blue.svg
+    :alt: License LGPL-3
+
+Purchase Request to RFQ Order Approved
+======================================
+
+This module computes the new PO state 'Approved' related to a Purchase
+Request Line to display it in the Purchase Request Line tree view and adds a
+'Purchase Approved' filter.
+
+Usage
+=====
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/repo/github-com-oca-purchase-workflow-142
```

Comment:
the correct link is: https://runbot.odoo-community.org/runbot/142/9.0, just follow OCA template module